chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -1966,7 +1969,7 @@ private synchronized void saveOctree(RenderContext context, TaskTracker taskTrac
       task.update(1);
       Log.info("Saving octree " + fileName);
 
-      try (DataOutputStream out = new DataOutputStream(new GZIPOutputStream(context.getSceneFileOutputStream(fileName)))) {
+      try (DataOutputStream out = new DataOutputStream(new FastBufferedOutputStream(new GZIPOutputStream(context.getSceneFileOutputStream(fileName))))) {
```

Comment:
Shouldn't this be the other way around? Same for the three other lines below where a gzip stream is used.

```suggestion
      try (DataOutputStream out = new DataOutputStream(new GZIPOutputStream(new FastBufferedOutputStream(context.getSceneFileOutputStream(fileName))))) {
```

Reply:
Probably, though I'm not sure it matters, will profile it

Reply:
If [this](https://stackoverflow.com/questions/1082320/what-order-should-i-use-gzipoutputstream-and-bufferedoutputstream) is to be believed, wrapping the buffer around the gzip stream is the faster way (how we have it currently).